### PR TITLE
EmpoEx Full Implementation and OkCoin fix

### PR DESCRIPTION
**Pull request by zholmes1**

### CODE REVIEW

File: xchange-okcoin/pom.xml [hunk 1]
```diff
@@ -24,6 +24,12 @@
 	<!-- Parent provides default configuration for dependencies -->
 	<dependencies>
 		<dependency>
+			<groupId>com.e-movimento.tinytools</groupId>
+			<artifactId>privilegedaccessor</artifactId>
+			<version>1.2.2</version>
```
Comment (timmolter) [inline xchange-okcoin/pom.xml]: I'm curious what this is for. Never heard of it.

**Comment by timmolter** [comment]
@zholmes1 can you check on the merge conflicts? I guess you need to sync up with the latest develop commits?

**Comment by timmolter** [comment]
@zholmes1Any chance you could take a look at the merge conflicts?

**Comment by zholmes1** [comment]
I'm going to take this PR down and submit one with just my EmpoEX implementation.

https://github.com/timmolter/XChange/pull/820 fixes the OkCoin issue.

**Pull request closed without merging**